examples/templates/do-linux/main.tf

```diff
@@ -261,18 +267,18 @@ resource "coder_agent" "main" {
     script       = <<-EOT
       #!/bin/bash
       set -e
-      free -m | awk 'NR==2{printf "%.2f%%\t", $3*100/$2 }'
+      free -m | awk '/^Mem/ {printf "%.2f%%", $3*100/$2 }'
     EOT
   }
   metadata {
-    key          = "disk"
-    display_name = "Disk Usage"
+    key          = "home"
+    display_name = "Home Usage"
     interval     = 600 # every 10 minutes
     timeout      = 30  # df can take a while on large filesystems
     script       = <<-EOT
       #!/bin/bash
       set -e
-      df /home/coder | awk '$NF=="/"{printf "%s", $5}'
+      df -h /home/${data.coder_workspace.me.owner} | awk '/^\// { printf "%s/%s (%s)", $3, $2, $5 }'
     EOT
```

Comment:
This could be replaced with,

`script = "coder stat disk --path /home/${lower(data.coder_workspace.me.owner)}"`

Reply:
Changed and tested. Thanks I saw these somewhere and forgot about them. Works great!